installer/resources/pacbot_app/files/DB.sql

```diff
@@ -2209,9 +2209,9 @@ INSERT IGNORE INTO cf_RuleInstance (ruleId,ruleUUID,policyId,ruleName,targetType
+INSERT IGNORE INTO cf_RuleInstance (ruleId,ruleUUID,policyId,ruleName,targetType,assetGroup,alexaKeyword,ruleParams,ruleFrequency,ruleExecutable,ruleRestUrl,ruleType,ruleArn,status,userId,displayName,createdDate,modifiedDate,severity,category) VALUES ('Enable_Azure_DiagnosticSettings_Mandatory_Tags','Enable_Azure_DiagnosticSettings_Mandatory_Tags','Enable_Azure_DiagnosticSettings_Mandatory_Tags','Enable_Azure_DiagnosticSettings_Mandatory_Tags','subscription','Azure','MandatoryTagsDiagnosticSettings','{"params":[{"encrypt":false,"value":"check-if-diagnostic-setting-has-mandatory-tags","key":"ruleKey"},{"key":"failure","value":"Mandatory categories is NOT enabled for diagnostic settings","isValueNew":true,"encrypt":false},{"key":"SUCCESS","value":"Mandatory categories is enabled for Diagnostic settings","isValueNew":true,"encrypt":false},{"key":"policyName","value":"Include mandatory categories in the diagnostics setting","isValueNew":true,"encrypt":false},{"encrypt":false,"value":"medium","key":"severity"},{"encrypt":false,"value":"security","key":"ruleCategory"},{"encrypt":false,"value":"","key":"ruleOwner"}],"environmentVariables":[],"ruleId":"Enable_Azure_DiagnosticSettings_Mandatory_Tags","autofix":false,"alexaKeyword":"MandatoryTagsDiagnosticSettings","ruleRestUrl":"","targetType":"subscription","pac_ds":"azure","policyId":"Enable_Azure_DiagnosticSettings_Mandatory_Tags","assetGroup":"azure","ruleUUID":"Enable_Azure_DiagnosticSettings_Mandatory_Tags","ruleType":"ManageRule"}','0 0/6 * * ? *',NULL,NULL,'ManageRule','arn:aws:events:us-east-1:***REMOVED***:rule/Enable_Azure_DiagnosticSettings_Mandatory_Tags','ENABLED','ASGC','Configure Mandatory Categories in Diagnostics Setting','2022-09-26','2022-09-26','medium','security');
```

Comment:
Since now we are using the new cf_PolicyTable for policy details, can you please update these sql changes in cf-PolicyTable in DB_policy.sql

Reply:
Done with the changes